docs/incidents/incident-customize-incident-layout.md

```diff
@@ -115,4 +115,14 @@ Note: for the layout to be associated with incident/indicator type, the layout I
  }
  ```
  
- For example, see [Phishing incident layout](https://github.com/demisto/content/blob/master/Packs/Phishing/Layouts/layoutscontainer-Phishing.json).
\ No newline at end of file
+ For example, see [Phishing incident layout](https://github.com/demisto/content/blob/master/Packs/Phishing/Layouts/layoutscontainer-Phishing.json).
```

Comment:
Add a bit more space between the example and the header , maybe separator `---`

![image](https://user-images.githubusercontent.com/7270217/88277237-0b76c200-cce9-11ea-9498-36f6d4fd6c7a.png)